pkg/k8shandler/cluster.go

```diff
@@ -100,7 +100,7 @@ func (er *ElasticsearchRequest) CreateOrUpdateElasticsearchCluster() error {
 			}
 		}
 
-		er.UpdateClusterStatus()
+		_ = er.UpdateClusterStatus()
```

Comment:
do we need to be assigning the value to nothing?

Reply:
This is a go lint failure. If you want to ignore an error you have to blackhole the returned value so that it's explicitly clear that you're intentionally ignoring the returned response.